package.json

```
@@ -92,6 +92,6 @@
     "typescript": "3.3.1"
   },
   "peerDependencies": {
-    "react": "^16.8.0"
+    "react": "^16.8.1"
```

I guess we can keep it unchanged.
```suggestion
    "react": "^16.8.0"
```